shopfloor_reception_packaging_dimension/models/shopfloor_menu.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Camptocamp SA (http://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ShopfloorMenu(models.Model):
+    _inherit = "shopfloor.menu"
+
+    set_packaging_dimension_is_possible = fields.Boolean(
+        compute="_compute_set_packaging_dimension_is_possible"
+    )
+    set_packaging_dimension = fields.Boolean(
+        string="Set packaging dimension",
+        default=False,
+        help="If for the product being processed, its related packaging "
+        "dimension are not set, ask to fill them up.",
+    )
+
+    @api.depends("scenario_id")
+    def _compute_set_packaging_dimension_is_possible(self):
+        for menu in self:
+            menu.set_packaging_dimension_is_possible = menu.scenario_id.has_option(
+                "set_packaging_dimension"
+            )
```

Comment:
I wouldn't add options.
If you don't want this to be active, you uninstall the module.
Or is there some other need I'm missing?

Reply:
With the option, it allows to have multiple menus using the reception scenario and a different configuration.